SCAutolib/controller.py

```diff
@@ -197,6 +206,13 @@ def setup_system(self, install_missing: bool, gdm: bool, graphical: bool):
         dump_to_json(user.User(username="root",
                                password=self.lib_conf["root_passwd"]))
 
+        # Fedora requires python3-uinput from RPM and rsyslog
+        if os_version == OSVersion.Fedora:
+            run(['dnf', 'install', 'python3-uinput', 'rsyslog', '-y'])
```

Comment:
I tried to run the test in a fresh fedora VM and I found out that packages "openssl" and "nss-tools" (which are needed) were not installed. So could you please add them? Perhaps not here but in line 147 on controller where the general packages variable is defined.

Reply:
I am adding openssl and nss-tools to the controller location described.

I do see those packages listed in requirements.txt so, should I add those pip packages to a new section to install specific pip packages for graphical?

Reply:
Oh yeah, my bad. For some reason, they weren't installed when I ran the tests and I completely missed that they are there. Perhaps I did something wrong and did not install SCAutolib[graphical]. Let's consider it resolved.